lib/velveteen/worker.rb

```diff
@@ -64,15 +64,10 @@ def generate_schema(routing_key)
       File.expand_path(schema_file_path, Config.schema_directory)
     end
 
-    def message_schema
-      if self.class.message_schema
-        File.expand_path(self.class.message_schema, Config.schema_directory)
-      end
-    end
-
     def maybe_validate_message!
```

Comment:
Since message validation isn't optional anymore, "maybe" no longer accurately describes what's happening. Shall we remove that word?